### PR TITLE
Added public NEXTAUTH_URL.

### LINKED ISSUE #821: Invite Link address is missing
Image

### PR DESCRIPTION
**Pull request by splendidcomputer**
Closes #821

### CODE REVIEW

File: apps/web/frontend/components/inviteLink.tsx [hunk 1]
```diff
@@ -27,7 +27,11 @@ export const InviteLink = (props: InviteLinkProps) => {
   const [{ fetching: fetchingRegenerateInviteKey }, regenerateInviteKey] = useMutation(
     projectRegenerateInviteKeyMutation,
   )
-  const inviteLink = `${process.env.NEXTAUTH_URL}/projects/join/${project.inviteKey}`
+
+  // let nextAuthUrl = process.env.NEXTAUTH_URL || 'defaultFallbackURL'
```
Comment (moncapitaine) [inline apps/web/frontend/components/inviteLink.tsx:31]: remove

File: apps/web/frontend/components/inviteLink.tsx [hunk 2]
```diff
@@ -27,7 +27,11 @@ export const InviteLink = (props: InviteLinkProps) => {
...
+
+  const nextAuthUrl = process.env.NEXTAUTH_URL ?? process.env.NEXT_PUBLIC_APP_URL
```
Comment (moncapitaine) [inline apps/web/frontend/components/inviteLink.tsx:33]: only NEXT_PUBLIC_APP_URL

File: apps/web/frontend/components/inviteLink.tsx [hunk 3]
```diff
@@ -27,7 +27,9 @@ export const InviteLink = (props: InviteLinkProps) => {
   const [{ fetching: fetchingRegenerateInviteKey }, regenerateInviteKey] = useMutation(
     projectRegenerateInviteKeyMutation,
   )
-  const inviteLink = `${process.env.NEXTAUTH_URL}/projects/join/${project.inviteKey}`
+
+  const nextAuthUrl = process.env.NEXT_PUBLIC_APP_URL
+  const inviteLink = `${nextAuthUrl}/projects/join/${project.inviteKey}`
```
Comment (PascalHelbig) [inline apps/web/frontend/components/inviteLink.tsx:32]: Now, it no longer related to next auth

```suggestion
  const inviteLink = `${process.env.NEXT_PUBLIC_APP_URL}/projects/join/${project.inviteKey}`
```

**Review by PascalHelbig — APPROVED** [review]
excited-the-child-gif-by-disney

**Pull request merged**